app/models/teleconsultation_medical_officer.rb

```diff
@@ -0,0 +1,2 @@
+class TeleconsultationMedicalOfficer < User
+end
```

Comment:
Can we set up the associations without this subclass? It seems unnecessary 🤔

Reply:
I added this class for encapsulating some of the serialization-deserialization logic for the MO sync endpoint. This is somewhat premature but will come handy very soon.